src/fetch/transfer_file.py

```diff
@@ -108,6 +110,18 @@ def from_dict(cls, obj: dict[str, str]) -> Transfer:
             amount_wei=int(obj["amount"]),
         )
 
+    @classmethod
+    def from_dataframe(cls, pdf: pd.DataFrame) -> list[Transfer]:
+        """Converts Pandas Dataframe into list of Transfers"""
+        return [
+            cls(
+                token=Token(row["token_address"]) if row["token_address"] else None,
```

Comment:
I'm pretty sure missing values are turned into `None` automatically. But I couldn't find any documentation specifically for `pd.read_sql` and it seems to be missing the `na_values` parameter which is used in `pd.read_csv`

Reply:
yea... we could fix this later if that is indeed the case. Problem is passing None into the Token constructor here though.